main.go

```diff
@@ -762,6 +762,13 @@ func main() {
 		Programmer:    *programmer,
 	}
 
+	err := options.Verify()
```

Comment:
I am not sure whether this is the right place to put this, as we continue modifying the options after this.

What do you think @aykevl ?

Reply:
I had this doubt as well, I put it here because it looked to me that after this line nothing was setting GC, Scheduler or PrintSize.. very curious to see what's you thoughts are..

Reply:
Yes, I think it should be done when the `options` object is finished building and won't be modified anymore.

Looking down below I see that there is also a custom check for `panicStrategy`. I think it would make sense to move that one to the common `Verify` function too.

Reply:
@aykevl I moved panicStrategy check under options as well, and removed it from main.go

Reply:
@aykevl @jaddr2line I've also moved options.Validate call later in the code..
I'm still unsure about it, because a few lines later, in the "build" case of the switch, options.Target is modified.

Another option could be moving the option check into builder.NewConfig... 
what do you guys think?

Reply:
@aykevl @jaddr2line what do you think about moving the option check into builderNewConfig? 
The more I think about it, the more it looks to me the right place :thinking: